ulib/FStar.Math.Lib.fst

```diff
@@ -28,7 +28,10 @@ let slash_star_axiom a b c =
 
 val log_2: x:pos -> Tot nat
 let rec log_2 x =
-  if x >= 2 then 1 + log_2 (x / 2) else 0
```

Comment:
Changing this would break existing code that uses this function.
It'd be cleaner to define an alternative function that does what you want.